applications/machine_learning/README.rst

```diff
@@ -430,8 +416,8 @@ See :ref:`testing_rtt_connect` for detailed instructions about accessing the log
 
    You can also use ``rtt`` configuration to have the Thingy:53 use RTT for logs.
 
-Testing with Thingy devices
----------------------------
+Testing with Thingy:53
+----------------------
```

Comment:
Need to change to `Testing with Thingy:53`_ in line no:190. This will fix the doc build issue.